src/Parquet.Test/InheritedPropertiesTest.cs

```diff
@@ -0,0 +1,66 @@
+using System.Threading.Tasks;
+using System.IO;
+using Parquet.Data;
+using Parquet.Serialization;
+using Xunit;
+
+namespace Parquet.Test {
+  public class InheritedPropertiesTest : TestBase {
+    private InheritedClass[] GenerateRecordsToSerialize() {
+      InheritedClass record = new() {
+        BaseProperty = "A",
+        InheritedProperty = "B"
+      };
+
+      InheritedClass[] recordsToSerialize = new InheritedClass[1]
+      {
+        record
+      };
+
+      return recordsToSerialize;
+    }
+
+    [Fact]
+    public async Task Serialize_class_without_inherited_properties() {
+      InheritedClass[] recordsToSerialize = GenerateRecordsToSerialize();
+      Schema schema = SchemaReflector.Reflect<InheritedClass>();
+
+      // TODO: is there some `with/using` syntax i should use here?
+      MemoryStream stream = new();
+      await ParquetConvert.SerializeAsync(recordsToSerialize, stream, schema);
+      InheritedClass[] deserializedRecords = await ParquetConvert.DeserializeAsync<InheritedClass>(stream, fileSchema: schema);
```

Comment:
```suggestion
      MemoryStream stream = new();
      await ParquetConvert.SerializeAsync(recordsToSerialize, stream, schema);
      InheritedClass[] deserializedRecords = await ParquetConvert.DeserializeAsync<InheritedClass>(stream, fileSchema: schema);
```
opening up a convo around this syntax - i'm new to c#, so wasn't sure if i should be utilizing some `using` keyword with these variables.  let me know what the best practice is!